Assets/Scripts/Effects/Description.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Dynamic descriptions for list of effects
+// The intent is for cards to use the text 
+// in CardData.description unless they're on top
+// of the queue, when they use one of these.
+// This is a liiiittle bit of a cop-out to avoid 
+// caring about the context an effect appears in
+
+using System.Collections.Generic;
+using Effects;
+using Cards;
+
+// Describes damage, healing, armor, Deciever cost --
+// everything that happens in Effects.Effect
+public class EffectDescription
```

Comment:
Since you implemented the parser in this way (not what I expected). Is there any scene showing the update strings to check if it is working? According to Rider, none of the functions is actually called